packages/admin-stories/src/admin/table/GlobalErrorHandling.tsx

```diff
@@ -0,0 +1,74 @@
+import { Table, TableQuery, useTableQuery } from "@comet/admin";
+import { text } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as React from "react";
+
+import { apolloSwapiStoryDecorator } from "../../apollo-swapi-story.decorator";
+import { errorDialogStoryProviderDecorator } from "../../error-dialog-provider.decorator";
+
+interface People {
+    id: string;
+    name: string;
+    birthYear: string;
+    gender: string;
+    homeworld: {
+        name: string;
+    };
+}
+interface QueryData {
+    allPeople: {
+        people: People[];
+    };
+}
+
+const Story: React.FunctionComponent = () => {
+    const query = gql`
+        ${text("GQL Query", "query StarWarsPeople {allPeople { people { id name birthYear gender homeworld{ name } } }}")}
+    `;
+
+    const { tableData, api, loading, error } = useTableQuery<QueryData, {}>()(query, {
+        resolveTableData: (data) => ({
+            data: data.allPeople.people,
+            totalCount: data.allPeople.people.length,
+        }),
+        globalErrorHandling: true,
```

Comment:
Wouldn't it be better to explain the `globalErrorHandling` param in the props instead of creating a separate story?

Reply:
i don't know what you mean

Reply:
Sorry, I meant to write docs, not props 😆 

As I see it, the purpose of this story is to demonstrate the `globalErrorHandling` param, isn't it? I believe a section called "Error handling in table queries" in the MDX docs could be of great help to explain the behavior

Reply:
there are currently no docs which handle useTableQuery() and i don't want to add documentation for useTableQuery now. I think we better stick with this story for now